### PR TITLE
fix example code; add explanation; minor grammar/formatting fixes

### PR DESCRIPTION
**Pull request by gj**
* remove useless `if`/`else` statement in `posts#show` action
* add explanation of why we need an `if`/`else` for `posts#index` but not `posts#show`
* fix run-on
* fix comma splices
* fix comments in code snippets
* remove dupe Learn link

### CODE REVIEW

**Comment by gj** [comment]
Covers: https://github.com/learn-co-curriculum/routing-nested-resources-reading/pull/9

Fixes: https://github.com/learn-co-curriculum/routing-nested-resources-reading/issues/5, https://github.com/learn-co-curriculum/routing-nested-resources-reading/issues/10

File: README.md [hunk 1]
```diff
@@ -136,25 +137,21 @@ Let's update `index` and `show` for the new routes:
...
-    end
+    @post = Post.find(params[:id])
   end
-
-  def new
-    # ...
 ```
 
-We didn't have to make any new methods, or make any calls to render templates explicitly, just add a simple check for `params[:author_id]` to each method and go from there.
+We added a conditional to the `posts#index` action to account for whether the user is trying to access the index of *all* posts (`Post.all`) or just the index of all posts *by a certain author* (`Author.find(params[:author_id]).posts`). The conditional hinges on whether there's an `:author_id` key in the `params` hash — in other words, whether the user navigated to `/authors/:id/posts` or simply `/posts`. We didn't have to create any new methods or make explicit calls to render new templates. We just added a simple check for `params[:author_id]`, and we're good to go.
```
Comment (gj) [inline README.md]: @mendelB new explanation on lines 144 & 148

**Comment by gj** [comment]
Also, this 'Lab' passes w/o modifying any code. Maybe another Readme posing as a Lab? @AnnJohn

**Comment by AnnJohn** [comment]
@gj changed this to a Readme and assigned the PR to @mendelB to review first. thanks!

**Comment by mendelB** [comment]
Stellar work on this @gj! merging.

**Pull request merged**